tests/integration/delta_lake/test_table_read.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import datetime
+
+import pandas as pd
+import pytest
+
+deltalake = pytest.importorskip("deltalake")
```

Comment:
you need to add `deltalake` to our requirements-dev and to our pyproject.toml (pip install getdaft[deltalake]) otherwise this test will never be run.